### PR TITLE
Add Japanese specific plan for land mobile station.

### PR DESCRIPTION
**Pull request by tooruuetani**
#### Summary
Add Japanese specific plan for land mobile station.
Using from 920.6MHz to 923.4MHz with LBT.

A land mobile station can transmit at MAX EIRP 27 dBm.
(Note) A user who installs land mobile station in Japan must apply to Japanese Ministry of Internal Affairs and Communications.

References #7, 
See https://github.com/TheThingsNetwork/lorawan-frequency-plans/pull/36 .

#### Changes
Currently AS_920_923 plan defines MAX EIRP 16 dBm, but some users want to use MAX EIRP 27 dBm for land mobile station.
It is compliant to Japanese law ARIB-STD-T108, See attached table. 

https://www.arib.or.jp/kikaku/kikaku_tushin/std-t108.html

#### Notes for Reviewers
Second plan "AS_920_923_JP_TTN_3_LAND_MOBILE" is for 16 channels gateway, so this plan is not intended to be used alone.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible, they don't break existing deployments.
- [x] Testing: The changes are tested.
- [x] Doc

### CODE REVIEW

File: frequency-plans.yml [hunk 1]
```diff
@@ -184,6 +184,20 @@
   country-codes: [jp, my, sg]
   file: lbt_80_over_128.yml
 
+- id: AS_920_923_JP_TTN_1_LAND_MOBILE
+  name: Japan 920-923 MHz with LBT (channels 31-38)
+  description: Frequency plan for Japanese land mobile station, using continuous frequencies up to 923.4MHz with MAX EIRP 27 dBm and LBT.
+  base-frequency: 915
+  country-codes: [jp]
+  file: AS_920_923_JP_TTN_1.yml
```
Comment (htdvisser) [inline frequency-plans.yml]:
```suggestion
  file: AS_920_923_JP_TTN_1_LAND_MOBILE.yml
```

File: frequency-plans.yml [hunk 2]
```diff
@@ -184,6 +184,20 @@
...
+
+- id: AS_920_923_JP_TTN_3_LAND_MOBILE
+  name: Japan 920-923 MHz with LBT (channels 24-31)
+  description: Frequency plan for Japanese land mobile station (16 channels), using continuous frequencies from 920.6 MHz with MAX EIRP 27 dBm and LBT, expected to be used with AS_920_923_JP_TTN_1.
+  base-frequency: 915
+  country-codes: [jp]
+  file: AS_920_923_JP_TTN_3.yml
```
Comment (htdvisser) [inline frequency-plans.yml]:
```suggestion
  file: AS_920_923_JP_TTN_3_LAND_MOBILE.yml
```

**Review by htdvisser — CHANGES_REQUESTED** [review]
Since compared to #36 this only changes the max-eirp, why can't we extend the "base" plans, and only override the sub-bands (like we do for `AS_920_923_LBT`)?

Then we'd only have to put this in the frequency plan files:

```yml
sub-bands:
  - min-frequency: 922000000
    max-frequency: 923400000
    max-eirp: 27
```

And in `frequency-plans.yml` we'd add:

```diff
 - id: AS_920_923_JP_TTN_3_LAND_MOBILE
   // ...
   base-frequency: 915
+  base-id: AS_920_923_JP_TTN_3
   country-codes: [jp]
   // ...
```

(BTW: here I'd also like to see the rename that I suggested in #36)

**Comment by htdvisser** [comment]
OK. #36 was just merged. If you address the open comments and re-request review (or ping me) we'll try to also get this one into the v3.19 release of The Things Stack.

**Comment by tooruuetani** [comment]
OK, Thanks !
I will update this PR later.

File: frequency-plans.yml (hunk 1 above)
Comment (tooruuetani) [reply frequency-plans.yml]: I fixed this.

File: frequency-plans.yml (hunk 2 above)
Comment (tooruuetani) [reply frequency-plans.yml]: I fixed this.

**Review by htdvisser — APPROVED** [review]
One suggestion for the `name`s, otherwise this looks good to go.

**Review by adriansmares — COMMENTED** [review]
Nothing to add - LGTM.

**Pull request merged**